app/rtkrcv/rtk_skylark_l1l5.conf

```diff
@@ -0,0 +1,151 @@
+out-outhead        =off        # (0:off,1:on)
+out-outopt         =off        # (0:off,1:on)
+out-timesys        =gpst        # (0:gpst,1:utc,2:jst)
+out-timeform       =tow        # (0:tow,1:hms)
+out-timendec       =3
+out-degform        =dms        # (0:deg,1:dms)
+out-fieldsep       =
+out-outsingle      =off        # (0:off,1:on)
+out-maxsolstd      =0          # (m)
+out-height         =geodetic   # (0:ellipsoidal,1:geodetic)
+out-geoid          =internal   # (0:internal,1:egm96,2:egm08_2.5,3:egm08_1,4:gsi2000)
+out-solstatic      =all        # (0:all,1:single)
+out-nmeaintv1      =0          # (s)
+out-nmeaintv2      =0          # (s)
+out-outstat        =off        # (0:off,1:state,2:residual)
+stats-eratio1      =100
+stats-eratio2      =100
+stats-errphase     =0.01       # (m)
+stats-errphaseel   =0.01       # (m)
+stats-errphasebl   =0          # (m/10km)
+stats-errdoppler   =1          # (Hz)
+stats-stdbias      =30         # (m)
+stats-stdiono      =0.03       # (m)
+stats-stdtrop      =0.3        # (m)
+stats-prnaccelh    =10         # (m/s^2)
+stats-prnaccelv    =10         # (m/s^2)
+stats-prnbias      =0.0001     # (m)
+stats-prniono      =0.001      # (m)
+stats-prntrop      =0.0001     # (m)
+stats-prnpos       =0          # (m)
+stats-clkstab      =5e-12      # (s/s)
+ant1-postype       =llh        # (0:llh,1:xyz,2:single,3:posfile,4:rinexhead,5:rtcm,6:raw)
+ant1-pos1          =90         # (deg|m)
+ant1-pos2          =0          # (deg|m)
+ant1-pos3          =-6335367.6285 # (m|m)
+ant1-anttype       =
+ant1-antdele       =0          # (m)
+ant1-antdeln       =0          # (m)
+ant1-antdelu       =0          # (m)
+ant2-postype       =rtcm       # (0:llh,1:xyz,2:single,3:posfile,4:rinexhead,5:rtcm,6:raw)
+ant2-pos1          =0          # (deg|m)
+ant2-pos2          =0          # (deg|m)
+ant2-pos3          =0          # (m|m)
+ant2-anttype       =
+ant2-antdele       =0          # (m)
+ant2-antdeln       =0          # (m)
+ant2-antdelu       =0          # (m)
+ant2-maxaveep      =3600
+ant2-initrst       =on         # (0:off,1:on)
+misc-timeinterp    =off        # (0:off,1:on)
+misc-sbasatsel     =0          # (0:all)
+misc-rnxopt1       =
+misc-rnxopt2       =
+misc-pppopt        =
+file-satantfile    =
+file-rcvantfile    =
+file-staposfile    =
+file-geoidfile     =
+file-ionofile      =
+file-dcbfile       =
+file-eopfile       =
+file-blqfile       =
+file-tempdir       =
+file-geexefile     =
+file-solstatfile   =
+file-tracefile     =
+
+inpstr1-type       =ntripcli   # (0:off,1:serial,2:file,3:tcpsvr,4:tcpcli,7:ntripcli,8:ftp,9:http)
+inpstr2-type       =ntripcli   # (0:off,1:serial,2:file,3:tcpsvr,4:tcpcli,7:ntripcli,8:ftp,9:http)
+inpstr3-type       =off        # (0:off,1:serial,2:file,3:tcpsvr,4:tcpcli,7:ntripcli,8:ftp,9:http)
+inpstr1-path       =username:password@igs-ip.net:2101/STFU00USA0
+inpstr2-path       =username:password@na2.skylark.swiftnav.com:2101/CRS
```

Comment:
Add `note: host name may vary depending upon geographic region and frequency ranges`